contracts/abstract/MasterAwareV2.sol

```diff
@@ -8,10 +8,10 @@ import "../interfaces/IMemberRoles.sol";
 
 abstract contract MasterAwareV2 is IMasterAwareV2 {
 
-  mapping(uint => address payable) public internalContracts;
-
   INXMMaster public master;
 
+  mapping(uint => address payable) public internalContracts;
```

Comment:
this switch fixes all memory issues associated with the memory slot since LegacyMasterAware has the master address written in the same slot.